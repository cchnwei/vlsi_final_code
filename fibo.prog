

           
       
           
       
      
           
           
           
           
           
           
           
          
          
           
           
           
           
           
           
           
           
           
           
        
        
        
        
        

b7 92 00 00
13 05 f0 00
97 00 00 00
e7 80 c0 00
93 07 50 00
93 05 00 00
13 06 10 00
93 06 00 00
13 07 20 00
13 08 10 00
63 4c a8 00
63 06 05 01
13 05 00 00
ef 00 c0 02
13 05 10 00
ef 00 40 02
b3 86 c5 00
93 05 06 00
13 86 06 00
23 a0 d2 00
93 82 42 00
63 56 a7 00
13 07 17 00
ef f0 5f fe
13 05 06 00
b7 02 01 00
93 82 c2 ff
13 03 f0 ff
07 00 00 00 //+1
23 40 00 00 //
23 a0 62 00  
 

    
    
          
        
        
          
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
          
          
          
          
          
          
          
          
          
         
         
         
         
         
         
         
         
         
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
          
          
          
          
          
          
          
          
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           













































































































































































































