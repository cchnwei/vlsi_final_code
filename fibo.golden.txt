00000001
00000002
00000003
00000005
00000008
0000000d
00000015
00000022
00000037
00000059
00000090
000000e9
00000179
00000262
